controllers/provider-aws/charts/validator-aws/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
```

Comment:
The chart looks good, but it lacks the possibility to deploy it into a virtual garden environment. Could you please adapt it?

Reply:
It's now possible to pass a `kubeconfig` as well as an `url` as values, so that you can run it in a vg setup. Am I missing something?

Reply:
As discussed, I'll split the charts like in https://github.com/gardener/gardener/tree/master/charts/gardener/charts to allow a proper virtual Gardener setup